ui/components/app/edit-gas-popover/edit-gas-popover.component.js

```diff
@@ -83,11 +133,29 @@ export default function EditGasPopover({
     }
 
     closePopover();
-  }, [transaction, mode, dispatch, closePopover]);
+  }, [
+    transaction,
+    mode,
+    dispatch,
+    closePopover,
+    gasLimit,
+    gasPrice,
+    maxFeePerGas,
+    maxPriorityFeePerGas,
+    gasEstimateType,
+  ]);
+
+  let title = t('editGasTitle');
+  if (popoverTitle) {
+    title = popoverTitle;
```

Comment:
Just noting that the title priorities have changed here as well. Previously the education title would be shown over a custom title, but no longer.

This seems more appropriate though I think :thinking: